src/Console/ListenCommand.php

```diff
@@ -269,7 +266,7 @@ protected function setupWebhook(string $tunnel): void
                         'license_key_created',
                         'license_key_updated',
                     ],
-                    'secret' => config('lemon-squeezy.signing_secret'),
+                    'secret' => config('lemon-squeezy.signing_secret') ?: Str::limit(Hash::make(config('lemon-squeezy.api_key')), 32, '')
```

Comment:
Wondering if this isn't just simpler? 🤔 

```suggestion
                    'secret' => config('lemon-squeezy.signing_secret') ?: Str::random(32),
```

Reply:
https://github.com/lmsqueezy/laravel/blob/9cf73218924be59f1843e2e8c66434b0eba13c69/src/Http/Controllers/WebhookController.php#L41-L42

It wouldn't actually work since the incoming `LEMON_SQUEEZY_SIGNING_SECRET` won't match what is being looked for in the `VerifyWebhookSignature` middleware.

I tend to agree that perhaps this is not even really an issue, maybe just document that if using the listen command then ensure that a "dummy" `LEMON_SQUEEZY_SIGNING_SECRET` is be given?